docs/schema.md

```diff
@@ -0,0 +1,38 @@
+# AWS SAM JSON schema
+
+## Context
+
+The [AWS SAM specification](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/sam-specification.html) is defined by a [JSON schema](https://github.com/aws/serverless-application-model/discussions/2645).
+
+## Schema generation
+
+At a high level, the final [`schema.json`](https://github.com/aws/serverless-application-model/blob/develop/samtranslator/schema/schema.json) is generated as such:
+
+```mermaid
```

Comment:
It'd be great to see a screenshot with an actual diagram

Reply:
Click on the little document icon in the top bar